test/spec_set_x_forwarded_proto_header.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+
+
+describe Rack::SetXForwardedProtoHeader do
+  response = lambda {|e| [200, {}, []]  }
+
+  it "leaves the value of X_FORWARDED_PROTO intact if there is no vendor header passed in the request" do
+    vendor_forwarded_header = "not passed in the request"
+    env = Rack::MockRequest.env_for("/", "HTTP_X_FORWARDED_PROTO" => "http")
+
+    Rack::Lint.new(Rack::SetXForwardedProtoHeader.new(response, vendor_forwarded_header)).call env
+
+    env["HTTP_X_FORWARDED_PROTO"].must_equal "http"
+  end
+
+  it "returns early when there is no vendor header passed in the request" do
+    vendor_forwarded_header = "not passed in the request"
+    env = Rack::MockRequest.env_for("/", "FOO" => "bar")
+
+    header_middleware = Rack::SetXForwardedProtoHeader.new(response, vendor_forwarded_header)
+    # Patch to ensure we return early and do not call `copy_header_value`
+    def header_middleware.copy_header_value
```

Comment:
This method no longer exists.  Probably want to change the spec to test that `HTTP_X_FORWARDED_PROTO` is not set, with the spec description updated accordingly.

Reply:
Doh! Sorry about that.